From 303a27ce818a1da0dbaa56b3143129b3a2828191 Mon Sep 17 00:00:00 2001
From: Vinicius Costa Gomes <vinicius.gomes@intel.com>
Date: Wed, 24 Feb 2021 15:27:35 -0800
Subject: [PATCH 095/122] igc: Remove unneeded netif_trans_update() during
 reset

Removes the call to netif_trans_update() during update, it is not
needed and it only resets transmissions from queue 0.

Signed-off-by: Vinicius Costa Gomes <vinicius.gomes@intel.com>
Signed-off-by: Muhammad Husaini Zulkifli <muhammad.husaini.zulkifli@intel.com>
---
 drivers/net/ethernet/intel/igc/igc_main.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/net/ethernet/intel/igc/igc_main.c b/drivers/net/ethernet/intel/igc/igc_main.c
index a1502080c9cd..520a68833191 100644
--- a/drivers/net/ethernet/intel/igc/igc_main.c
+++ b/drivers/net/ethernet/intel/igc/igc_main.c
@@ -4453,9 +4453,6 @@ void igc_down(struct igc_adapter *adapter)
 	wr32(IGC_RCTL, rctl & ~IGC_RCTL_EN);
 	/* flush and sleep below */
 
-	/* set trans_start so we don't get spurious watchdogs during reset */
-	netif_trans_update(netdev);
-
 	netif_carrier_off(netdev);
 	netif_tx_stop_all_queues(netdev);
 
-- 
2.17.1

